IFC4x3/Properties/t/TS_3TIrUqMFzEHxl56bjVXyxc/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TS_3TIrUqMFzEHxl56bjVXyxc" Name="TS" UniqueId="dd4b57b4-58ff-4e47-bbc5-1a5b5f87cee6" PrimaryDataType="IfcReal" />
```

Comment:
What does TS mean?

Reply:
It says "The total sulfur content" in the description.